requests/test/src/requests/RequestTests.scala

```diff
@@ -191,16 +191,15 @@ object RequestTests extends TestSuite{
         compress = requests.Compress.Gzip,
         data = new RequestBlob.ByteSourceRequestBlob("I am cow")
       )
-      assert(read(new String(res2.bytes))("data").toString ==
-        """"data:application/octet-stream;base64,H4sIAAAAAAAAAPNUSMxVSM4vBwCAGeD4CAAAAA=="""")
+      assert(read(new String(res2.bytes))("data").toString.contains("data:application/octet-stream;base64,H4sIAAAAAA"))
 
       val res3 = requests.post(
         "https://httpbin.org/post",
         compress = requests.Compress.Deflate,
         data = new RequestBlob.ByteSourceRequestBlob("Hear me moo")
       )
-      assert(read(new String(res2.bytes))("data").toString == 
-        """"data:application/octet-stream;base64,H4sIAAAAAAAAAPNUSMxVSM4vBwCAGeD4CAAAAA=="""")
+      assert(read(new String(res3.bytes))("data").toString == 
```

Comment:
I believe this was supposed to be `res3`, not `res2` or else we're never doing anything with `res3`.